python/runtime/pai/submitter.py

```diff
@@ -653,7 +689,9 @@ def create_explain_result_table(datasource, data_table, result_table,
     db.execute(conn, drop_stmt)
 
     create_stmt = ""
-    if model_type == oss.MODEL_TYPE_TF:
+    if model_type == oss.MODEL_TYPE_PAIML:
```

Comment:
What do you thinking resuing  `EstimatorType` in `runtime.model` ?

Reply:
done